Makefile

```diff
@@ -6,8 +6,10 @@ ci-full: ci sec dependencies-analyze bench
 
 test: 
 	go test ./...
+	go test ./openapi3-testing
```

Comment:
Won't the second be already tested by the first one?

You can use `go test -v` to check locally

Reply:
No, `go get ./...` only execute tests for the module and its packages, not children modules

Reply:
OK, it's clearer